packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -231,50 +239,183 @@ export function buildWebpackBrowser(
+                  files.push(file);
+                  continue;
+                }
+
+                // Ignore already processed files; emittedFiles can contain duplicates
+                if (seen.has(file.file)) {
+                  continue;
+                }
+                seen.add(file.file);
+
+                // All files at this point except ES5 polyfills are module scripts
+                const es5Polyfills = file.file.startsWith('polyfills-es5');
+                if (!es5Polyfills && !file.file.startsWith('polyfills-nomodule-es5')) {
+                  moduleFiles.push(file);
+                }
+                // If not optimizing then ES2015 polyfills do not need processing
+                // Unlike other module scripts, it is never downleveled
+                if (!actionOptions.optimize && file.file.startsWith('polyfills-es2015')) {
+                  continue;
+                }
+
+                // Retrieve the content/map for the file
+                // NOTE: Additional future optimizations will read directly from memory
+                let filename = path.resolve(getSystemPath(root), options.outputPath, file.file);
+                const code = fs.readFileSync(filename, 'utf8');
+                let map;
+                if (actionOptions.sourceMaps) {
+                  try {
+                    map = fs.readFileSync(filename + '.map', 'utf8');
+                    if (es5Polyfills) {
+                      fs.unlinkSync(filename + '.map');
+                    }
+                  } catch {}
+                }
+
+                if (es5Polyfills) {
+                  fs.unlinkSync(filename);
+                  filename = filename.replace('-es2015', '');
+                }
+
+                // ES2015 polyfills are only optimized; optimization check was performed above
+                if (file.file.startsWith('polyfills-es2015')) {
+                  actions.push({
+                    ...actionOptions,
+                    filename,
+                    code,
+                    map,
+                    optimizeOnly: true,
+                  });
+
+                  continue;
+                }
+
+                // Record the bundle processing action
+                // The runtime chunk gets special processing for lazy loaded files
+                actions.push({
+                  ...actionOptions,
+                  filename,
+                  code,
+                  map,
+                  runtime: file.file.startsWith('runtime'),
+                });
+
+                // Add the newly created ES5 bundles to the index as nomodule scripts
+                const newFilename = es5Polyfills
+                  ? file.file.replace('-es2015', '')
+                  : file.file.replace('es2015', 'es5');
+                noModuleFiles.push({ ...file, file: newFilename });
+              }
+
+              // Execute the bundle processing actions
+              await new Promise<void>((resolve, reject) => {
```

Comment:
Can we print a message that something is happening? I found it a bit confusing that after webpack emitted its stats the process continued to run without outputting any information and this was on a minimal repo. At first I though it got stuck.

Reply:
Yes.  I’ll add it in.

Reply:
Done.